Sources/NIOCore/EventLoopFuture+AssumeIsolated.swift

```diff
@@ -25,9 +31,9 @@ struct IsolatedEventLoop {
     }
 
     /// Submit a given task to be executed by the `EventLoop`
+    @available(*, noasync)
     @inlinable
-    func execute(_ task: @escaping () -> Void) {
-        self._wrapped.assertInEventLoop()
```

Comment:
Meta question: I understand why you removed those but should we keep them in in-case somebody really mishandles this type? In release builds these all should go away anyways.